BP/history/provinces/bp_k_septem_provincae.txt

```diff
@@ -1,40 +1,68 @@
-﻿#k_septem_provincae
+﻿# Notes:
+# Aqutitanian language extinct by Early Middle Ages (so ca. 700 by the Mod's reckoning)
+#
+# Christian sites in AD 325:
+# https://en.wikipedia.org/wiki/History_of_Christianity#/media/File:Spread_of_Christianity_to_AD_600_(1).png
+#
+# Manicheanism died out in the west by the close of the 5th century, and was nearly obliterated in the east in the Sixth
+# but a small core remained in the east as late as the ninth century.
+# "Merriam-Webster's Encyclopedia of World Religions". Merriam-Webster. pp. 689-90
+# https://www.tertullian.org/fathers/photius_copyright/photius_04bibliotheca.htm#179
+#
+# To tie into the eventual development of Catharism in Occitania and taking into account the early spread of Christianity
+# Manicheanism is instead focused in the Beziers/Carcassonne region where friction between Arians and Orthodox were
+# the highest (which would have given it breathing room).
+#
+# Reduced Visigothic presence to be more focused on the Garonne river as per their historical settlement in 395
```

Comment:
Replaced by Romano-Hispanic? I don't see the justification for that in the Seven Provinces.

Reply:
In WtWSMS, Romano-Hispanic was very tightly coupled with Manicheanism; As such, I kept the Manichean and Romano-Hispanic provinces tightly coupled as well. I presume the Romano-Hispanic presence should be tied to the eventual rise of Occitan culture as well and south d_toulouse/d_narbonensis_prime is the heartland of the Languedocien languages (which are more related to Catalan/Valencian than langues d'oïl.

Reply:
> In WtWSMS, Romano-Hispanic was very tightly coupled with Manicheanism; As such, I kept the Manichean and Romano-Hispanic provinces tightly coupled as well. I presume the Romano-Hispanic presence should be tied to the eventual rise of Occitan culture as well and south d_toulouse/d_narbonensis_prime is the heartland of the Languedocien languages (which are more related to Catalan/Valencian than langues d'oïl.

Occitan does not come from influence from Iberia, but rather a different influence from Latin and Romano-Gallic, with a much weaker Germanic influence. With this in mind it does not make sense to have those Romano-Hispanic provinces. If Gallo-Roman is to be less present then it is rather Roman proper which should replace it. You can't speak of an Iberian influence on the Seven Provinces. By the way, in WtWSMS only Gallo-Roman can turn into Occitan, and the presence of Romano-Hispanic is reduced to areas actually influenced by the peninsula. So if you want to extend Manichean it can not be coupled with Romano-Hispanic.

Reply:
Revised with 82b6a22; includes new Roman provinces on the Rhone and Provence and removes Romano-Hispanic in Septem Provincae in favor of Romano-Gallic. If acceptable, mark as resolved.

Reply:
Ok